src/appleseed/renderer/kernel/rendering/final/uniformpixelrenderer.h

```diff
@@ -53,7 +53,7 @@ namespace renderer
 //
 
 class UniformPixelRendererFactory
-  : public PixelRendererBaseFactory
+  : public IPixelRendererFactory
```

Comment:
Nobody is inheriting from `PixelRendererBase` anymore? That's unexpected. The point of `PixelRendererBase` was to provide default (empty) implementations for a number of methods. However if truly no pixel renderer needs it then it needs to be removed.

Reply:
It's just the factory. It was like this before I implemented the save_diagnostic PR months ago.

Reply:
Oh right, thanks.